exec/jdbc-all/pom.xml

```diff
@@ -452,6 +452,7 @@
                <exclude>org/apache/drill/shaded/guava/com/google/common/graph/**</exclude>
                <exclude>org/apache/drill/shaded/guava/com/google/common/collect/Tree*</exclude>
                <exclude>org/apache/drill/shaded/guava/com/google/common/collect/Standard*</exclude>
+               <exclude>org/apache/drill/shaded/guava/com/google/common/io/BaseEncoding*</exclude>
```

Comment:
Why was this needed?

Reply:
The JDBC jar file was exceeding size limits.  We have done this type of exclusions in the past also.